docs/changelog/98138.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 98138
+summary: Do not store default template with search application if it is not sp…
```

Comment:
Do we want to create a changelog with this, or should we tag the PR with `>non-issue`?

Reply:
That's a good question. It is a change to a product that's already released, but it's also beta. I could go either way.